robottelo/cli/factory.py

```diff
@@ -2362,7 +2363,10 @@ def setup_cdn_and_custom_repos_content(
         # Upload the organization manifest
         try:
             manifests.upload_manifest_locked(
-                org_id, manifests.clone(), interface=manifests.INTERFACE_CLI
```

Comment:
@mshriver we should use `with` statement whenever we are doing `manifests.clone()`, `manifests.upload_manifest_locked` or any such operation, right?
I am not sure, but could this be the reason we are having issue with manifest?